app/code/Magento/Inventory/Controller/Adminhtml/Sources/NewAction.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Controller\Adminhtml\Block;
+
+class NewAction extends \Magento\Cms\Controller\Adminhtml\Block
+{
+    /**
+     * @var \Magento\Backend\Model\View\Result\ForwardFactory
+     */
+    protected $resultForwardFactory;
```

Comment:
Magento does not encourage inheritance-based API. This method should be private.